### PR TITLE
Update _card.scss

### PR DESCRIPTION
**Pull request by ykris45**
Invites section doesn't expand past a certain height
https://media.discordapp.net/attachments/363087331475062785/493920826505035776/unknown.png?width=693&height=428
Displays correctly when position is set from relative to initial in the CSS.

### CODE REVIEW

File: src/renderer/scss/component/_card.scss [hunk 1]
```diff
@@ -4,7 +4,7 @@
   border-radius: var(--card-radius);
   user-select: text;
   display: flex;
-  position: relative;
+  position: initial;
```
Comment (btzr-io) [inline src/renderer/scss/component/_card.scss]: maybe if you just remove this line should also work?

Comment (seanyesmunt) [reply src/renderer/scss/component/_card.scss]: That would be ideal if it works.

Comment (ykris45) [reply src/renderer/scss/component/_card.scss]: @seanyesmunt yes it works

Comment (ykris45) [reply src/renderer/scss/component/_card.scss]: image

**Comment by seanyesmunt** [comment]
@ykris45 We use the `.card` class in a lot of places. Have you tested that this doesnt affect other parts of the app? I will give it a test in a little bit.

**Comment by ykris45** [comment]
@seanyesmunt i doesnt have a lot of referalls. and i will test it on locally.

**Pull request merged**